app/pb_kits/playbook/pb_selectable_card_icon/_selectable_card_icon.scss

```diff
@@ -0,0 +1,65 @@
+@import "../tokens/colors";
```

Comment:
In general, this seems to have more spacing than other files. Do you have 4 spaces for tabs? I believe 2 is what most folks have

Reply:
Changed VS Code settings to 2 spaces.  Cleaned up Sass files indentation.  Will clean up rails/ruby as well.